### ISSUE TITLE
Save all sources is doing nothing

### ISSUE THREAD

**Issue by pankajtt** [java-decompiler/jd-gui#14]
i am able to save the class files one by one but save all sources is not doing anything.

**Comment by emmanue1** [comment]
It works for me. Did you take a look on issue #7? What are your Java version, OS and architecture ?

**Comment by 4ndrew** [comment]
@pankajtt I had the same problem with save all, solved by installing recent java 1.8.0_45

**Comment by l29ah** [comment]
Encountering this on
```
‰ java -version 
openjdk version "1.8.0_151"
OpenJDK Runtime Environment (IcedTea 3.6.0) (Gentoo icedtea-3.6.0)
OpenJDK 64-Bit Server VM (build 25.151-b12, mixed mode)
```
I see this in the console meanwhile:
```
ConstantPool.addConstantClass: invalid name index
ConstantPool.addConstantClass: invalid name index
ConstantPool.addConstantClass: invalid name index
ConstantPool.addConstantClass: invalid name index
ConstantPool.addConstantClass: invalid name index
ConstantPool.addConstantClass: invalid name index
ConstantPool.addConstantClass: invalid name index
ConstantPool.addConstantClass: invalid name index
ConstantPool.addConstantClass: invalid name index
ConstantPool.addConstantClass: invalid name index
ConstantPool.addConstantClass: invalid name index
'AnnotatedElementDescriptor.createAnnotation' build error: stack not empty. stack=[jd.core.model.instruction.bytecode.instruction.DupLoad@3cf31c67]
'AnnotatedElementDescriptor.getProxyClassLoader' build error: stack not empty. stack=[jd.core.model.instruction.bytecode.instruction.DupLoad@392b0e7e, jd.core.model.instruction.bytecode.instruction.ALoad@3736fbdb]
'AnnotatedElementDescriptor.getEnumValue' build error: stack not empty. stack=[jd.core.model.instruction.bytecode.instruction.ALoad@77475ac3]
'AnnotationHandler.invoke' build error: stack not empty. stack=[jd.core.model.instruction.bytecode.instruction.AALoad@53f37068]
'SignatureList.getMap' build error: stack not empty. stack=[jd.core.model.instruction.bytecode.instruction.DupLoad@324adc96]
'WildcardTypeImpl.<clinit>' build error: stack not empty. stack=[jd.core.model.instruction.bytecode.instruction.DupLoad@32220127]
'ElementType_.valueOf' build error: stack not empty. stack=[jd.core.model.instruction.bytecode.instruction.DupLoad@7f7ed4e2]
'RetentionPolicy_.valueOf' build error: stack not empty. stack=[jd.core.model.instruction.bytecode.instruction.DupLoad@37c68b55]
'Enum_.getDeclaringClass' build error: stack not empty. stack=[jd.core.model.instruction.bytecode.instruction.ALoad@538c4f57]
'Enum_.getMap' build error: stack not empty. stack=[jd.core.model.instruction.bytecode.instruction.Invokevirtual@71854738]
'_Class.isEnum' build error: stack not empty. stack=[jd.core.model.instruction.bytecode.instruction.Invokevirtual@7855442e]
'AbstractCellEditor.addCellEditorListener' build error: stack not empty. stack=[jd.core.model.instruction.bytecode.instruction.GetField@7bcae452]
'AbstractCellEditor.removeCellEditorListener' build error: stack not empty. stack=[jd.core.model.instruction.bytecode.instruction.GetField@4a8dfa8b]
'JTreeTable.<init>' build error: stack not empty. stack=[jd.core.model.instruction.bytecode.instruction.ALoad@11eac865, jd.core.model.instruction.bytecode.instruction.ALoad@43cc9a42]
'JTreeTable.getEditingRow' build error: stack not empty. stack=[jd.core.model.instruction.bytecode.instruction.Invokevirtual@6f881e1d]
'MyTreeTable.resetTree' build error: stack not empty. stack=[jd.core.model.instruction.bytecode.instruction.ALoad@4af4e8c5, jd.core.model.instruction.bytecode.instruction.ALoad@d9aa81e]
'VariableModel.<clinit>' build error: stack not empty. stack=[jd.core.model.instruction.bytecode.instruction.DupLoad@2a232d65, jd.core.model.instruction.bytecode.instruction.DupLoad@574a6ca]
'Main.main' build error: stack not empty. stack=[jd.core.model.instruction.bytecode.instruction.GetStatic@682bd461]
'Main.badUsage' build error: stack not empty. stack=[jd.core.model.instruction.bytecode.instruction.GetStatic@30a94b22]
'FlexibleCompletor.<init>' build error: stack not empty. stack=[jd.core.model.instruction.bytecode.instruction.ALoad@3d21057, jd.core.model.instruction.bytecode.instruction.ALoad@2330b411]
'Global.init' build error: stack not empty. stack=[jd.core.model.instruction.bytecode.instruction.ALoad@150ee4ca]
'Global.defineClass' build error: stack not empty. stack=[jd.core.model.instruction.bytecode.instruction.DupLoad@6f045d12]
'Global.loadClass' build error: stack not empty. stack=[jd.core.model.instruction.bytecode.instruction.DupLoad@7c47a80d]
'JavaPolicySecurity.getStaticSecurityDomainClassInternal' build error: stack not empty. stack=[jd.core.model.instruction.bytecode.instruction.DupLoad@5a710ba8]
'Main.processOptions' build error: stack not empty. stack=[jd.core.model.instruction.bytecode.instruction.Invokevirtual@38bb8c98]
'XML.findAnnotation' build error: stack not empty. stack=[jd.core.model.instruction.bytecode.instruction.ALoad@751aaa50]
'XMLLibImpl.init' build error: stack not empty. stack=[jd.core.model.instruction.bytecode.instruction.DupLoad@345c633e]
'XmlNode.<clinit>' build error: stack not empty. stack=[jd.core.model.instruction.bytecode.instruction.DupLoad@66220256]
'ContextFactory.isDom3Present' build error: stack not empty. stack=[jd.core.model.instruction.bytecode.instruction.ALoad@2199d46]
'JavaAdapter.loadAdapterClass' build error: stack not empty. stack=[jd.core.model.instruction.bytecode.instruction.ALoad@22f1ef7f, jd.core.model.instruction.bytecode.instruction.ALoad@75633d8c]
'NativeIterator.jsConstructor' build error: stack not empty. stack=[jd.core.model.instruction.bytecode.instruction.Invokevirtual@325a1c4]
'PolicySecurityController.getStaticSecurityDomainClassInternal' build error: stack not empty. stack=[jd.core.model.instruction.bytecode.instruction.DupLoad@4d98ef4b]
'PolicySecurityController.loadBytecode' build error: stack not empty. stack=[jd.core.model.instruction.bytecode.instruction.DupLoad@60bf6b90]
'3.run' build error: stack not empty. stack=[jd.core.model.instruction.bytecode.instruction.ALoad@cdcb556]
'ScriptRuntime.<clinit>' build error: stack not empty. stack=[jd.core.model.instruction.bytecode.instruction.DupLoad@22590e88]
'2.run' build error: stack not empty. stack=[jd.core.model.instruction.bytecode.instruction.ALoad@28b15f3f]
```
